src/main/java/models/User.java

```diff
@@ -11,7 +11,10 @@ public User(){
     }
 
     public boolean willHit(){
-        return true;
-    };
```

Comment:
ah. that makes sense.
I was thinking we can use this to auto-stay once (user.sum >=21);